### ISSUE TITLE
[Bug] Fisherman Table

### ISSUE THREAD

**Issue by Tetrajak** [crazysnailboy/VillagerTrades#12]
Just a small bug; the fisherman's trade table includes the removal of a bow. I assume this is instead supposed to remove the enchanted fishing rod at this level?

**Comment by crazysnailboy** [comment]
Yes that's correct - the intention was to move the enchanted fishing rod trade to level 3 and replace the level 2 trade with a normal fishing rod. Thanks for pointing it out, I'll get it fixed :)
